### PR TITLE
Create parent directory when generating keys

### PR DESCRIPTION
**Pull request by Leland-Takamine**

### CODE REVIEW

File: dadb/src/main/kotlin/dadb/AdbKeyPair.kt [hunk 1]
```diff
@@ -100,6 +100,9 @@ class AdbKeyPair(
                 it.genKeyPair()
             }
 
+            privateKeyFile.absoluteFile.parentFile?.mkdirs()
+            publicKeyFile.absoluteFile.parentFile?.mkdirs()
```
Comment (amanjeetsingh150) [inline dadb/src/main/kotlin/dadb/AdbKeyPair.kt:104]: This is to generate .android folder if it doesn't exist?

Comment (Leland-Takamine) [reply dadb/src/main/kotlin/dadb/AdbKeyPair.kt:104]: Yeah exactly

Comment (amanjeetsingh150) [reply dadb/src/main/kotlin/dadb/AdbKeyPair.kt:104]: Got it, don't we need to pass the name of directory as .android?

**Review by amanjeetsingh150 — APPROVED** [review]

File: dadb/src/main/kotlin/dadb/AdbKeyPair.kt (hunk 1 above)
Comment (Leland-Takamine) [reply dadb/src/main/kotlin/dadb/AdbKeyPair.kt:104]: This is a generic method that can generate the key to any location. This logic ensure that no matter where you point to, the parent directories will be created if they don't already exist

Comment (amanjeetsingh150) [reply dadb/src/main/kotlin/dadb/AdbKeyPair.kt:104]: Ahhh gotcha!

**Pull request merged**